packages/astro/src/@types/astro.ts

```diff
@@ -462,6 +462,24 @@ export interface AstroUserConfig {
 	 */
 	site?: string;
 
+
+	/**
+	 * @docs
+	 * @name compressHTML
+	 * @type {boolean}
+	 * @default `false`
+	 * @description
+	 * When enabled Astro removes all whitespace from your final HTML, including line breaks. This is useful for reducing the size of HTML transfered over the wire in the development and the production.
```

Comment:
```suggestion
	 * This is an option to minify your HTML output and reduce the size of your HTML files. When enabled, Astro removes all whitespace from your HTML, including line breaks, from `.astro` components. This occurs both in development mode and in the final build.
```
It felt like the changeset had more (and important!) details than the actual docs did.  :smile: Would something like this be correct?

Reply:
Yes!